Makefile

```diff
@@ -55,6 +55,19 @@ dnsrecord-plan: dnsrecord-init
 dnsrecord-apply: dnsrecord-init
 	cd dns/records && terraform apply -var-file workspace-variables/${DNS_ZONE}_${DNS_ENV}.tfvars.json ${AUTO_APPROVE}
 
+sqlpad-init: set-azure-account
+	$(if $(or $(DISABLE_PASSCODE),$(PASSCODE)), , $(error Missing environment variable "PASSCODE", retrieve from https://login.london.cloud.service.gov.uk/passcode))
+	cd sqlpad && terraform init -backend-config workspace_variables/backend_${DEPLOY_ENV}.tfvars -upgrade -reconfigure
```

Comment:
Instead of `cd sqlpad` we could use the fancy [chdir option](https://www.terraform.io/cli/commands#switching-working-directory-with-chdir)

Reply:
changed to use chdir